superset-frontend/plugins/plugin-chart-pivot-table/src/react-pivottable/TableRenderers.jsx

```diff
@@ -20,12 +20,15 @@
 import React from 'react';
 import { t } from '@superset-ui/core';
 import PropTypes from 'prop-types';
-import { t } from '@superset-ui/core';
 import { PivotData, flatKey } from './utilities';
 import { Styles } from './Styles';
 
 const parseLabel = value => {
```

Comment:
We do not need to change the parseLabel function. Instead, we can simply translate the METRIC_KEY constant in superset-frontend/plugins/plugin-chart-pivot-table/src/PivotTableChart.tsx:58